README.md

```diff
@@ -126,4 +128,10 @@ If you have any code contributions in mind, please feel free to say Hi to us on
 # Disclaimer
 The authors of this project are not responsible for any content generated using this interface.
 
-The license of this software forbids you from sharing any content that violates any laws, produce any harm to a person, disseminate any personal information that would be meant for harm, spread misinformation, or target vulnerable groups. For the full list of restrictions please read [the license](LICENSE). You agree to these terms by using this software.
+The license of this software forbids you from: 
```

Comment:
The license of this software forbids you from sharing any content that:
- violates any laws.
- produces any harm to a person
- ...

There's a subtle but important legal difference.

Reply:
Changed to: The license of this software forbids you from sharing any content that:
- Violates any laws.
- Produces any harm to a person or persons.
- Disseminates (spreads) any personal information that would be meant for harm.
- Spreads misinformation.
- Target vulnerable groups.